lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
Well, `{ data: [ { correlationIds: [1,2] } ], ... }` isn't too bad to me, and it is far more simpler than other proposed approach. Besides, the only case where the `data` array contains more than `correlationIds` is when `unsubscribe` request come in and there are some unpolled buffer data, so we want to send back the last chunk. This should be a relative rare case.